install/Knative-with-OpenShift.md

```diff
@@ -11,168 +11,154 @@ You can find [guides for other platforms here](README.md).
+minishift config set image-caching true
+minishift addon enable admin-user
+minishift addon enable anyuid
 ```
 
-## Scripted cluster setup and installation
-
-For Linux and Mac, you can optionally run a
-[script](scripts/knative-with-openshift.sh) that automates the steps on this
-page.
+The above configuration ensures that Knative gets created in its own [minishift profile](https://docs.okd.io/latest/minishift/using/profiles.html) with 8GB of RAM and 4 vCpus. The image-caching helps in re-starting up the cluster faster every time.  The [addon](https://docs.okd.io/latest/minishift/using/addons.html) **admin-user** creates a default `admin` user with the role  cluster-admin and the  [addon](https://docs.okd.io/latest/minishift/using/addons.html) **anyuid** allows the `default` service account to run the application with uid `0`.
 
-Once you have `oc` present on your machine and in your `PATH`, you can simply
-run [this script](scripts/knative-with-openshift.sh); it will:
+To start minishift:
 
-- Create a new OpenShift cluster on your local machine with `oc cluster up`
-- Install Istio and Knative serving
-- Log you in as the cluster administrator
-- Set up the default namespace for istio autoinjection
+```shell
+minishift profile set knative
+minishift start
+```
 
-Once the script completes, you'll be ready to test out Knative!
+The command `minishift profile set knative` is required every time you start and stop minishift to make sure that you are on right `knative` minishift profile that was configured above.
 
-## Creating a new OpenShift cluster
+## Configuring `oc` (openshift cli)
 
-Create a new OpenShift cluster on your local machine using `oc cluster up`:
+Running the following command make sure that we have right version of `oc` and configured the DOCKER daemon to be connected to minishift Docker.
 
 ```shell
-oc cluster up --write-config
-
-# Enable admission webhooks
-sed -i -e 's/"admissionConfig":{"pluginConfig":null}/"admissionConfig": {\
-    "pluginConfig": {\
-        "ValidatingAdmissionWebhook": {\
-            "configuration": {\
-                "apiVersion": "v1",\
-                "kind": "DefaultAdmissionConfig",\
-                "disable": false\
-            }\
-        },\
-        "MutatingAdmissionWebhook": {\
-            "configuration": {\
-                "apiVersion": "v1",\
-                "kind": "DefaultAdmissionConfig",\
-                "disable": false\
-            }\
-        }\
-    }\
-}/' openshift.local.clusterup/kube-apiserver/master-config.yaml
-
-oc cluster up --server-loglevel=5
+eval $(minishift docker-env) && eval $(minishift oc-env)
 ```
 
-Once the cluster is up, login as the cluster administrator:
+## Preparing Knative Deployment
+
+### Enable Admission Controller Webhook
+To be able to deploy and run serverless Knative applications, its required that we enable the [Admission Controller Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/).  
```

Comment:
"its required that we enable" --> "you must enable"

Reply:
done